integration-tests/utils/integration-helpers.js

```diff
@@ -184,6 +207,26 @@ const deleteRuntimes = async ({ page, billingProject, workspaceName }) => {
   console.info(`deleted runtimes: ${deletedRuntimes}`);
 };
 
+const deleteRuntimesV2 = async ({ page, billingProject, workspaceName }) => {
+  const deletedRuntimes = await page.evaluate(
+    async (workspaceName) => {
+      const {
+        workspace: { workspaceId },
+      } = await window.Ajax().Workspaces.workspaceV2(billingProject, workspaceName).details(['workspace.workspaceId']);
```

Comment:
It looks like `details` does not exist on `workspaceV2`?

https://github.com/DataBiosphere/terra-ui/blob/c3251ba48ba30d6e642079beb9f3c599403b826e/src/libs/ajax.js#L130-L143